src/ConfigHelper.ts

```diff
@@ -133,17 +135,23 @@ export function addWhitelistedUIProps(props) {
 }
 
 const PROCESSED_VIEW_NAMES = new Set();
+
+interface ViewConfig {
+  uiViewClassName: string;
+  validAttributes: Record<string, any>;
```

Comment:
What `validAttributes` represent?

Reply:
properties of given `viewConfig` that should be added to props whitelist.

Reply:
so `Record<string, boolean>`?

Reply:
I guess there might be other types than `boolean` as well, see: https://github.com/facebook/react-native/blob/main/Libraries/Components/ScrollView/ScrollViewViewConfig.js#L32. 
I think the best option will be `Record<string, unknown>` then.